gtsam/hybrid/tests/testMixtureFactor.cpp

```diff
@@ -115,6 +118,156 @@ TEST(MixtureFactor, Dim) {
   EXPECT_LONGS_EQUAL(1, mixtureFactor.dim());
 }
 
+/* ************************************************************************* */
+// Test components with differing means
```

Comment:
I need more explanation in the comment to understand these tests. Also, since in the end they are just regression tests, it’s unclear they assert correctness of anything :-/ if these examples are supposed to be equivalent to the HBN tests above, and you are using the same values as there to test the result, that is a different story. Unclear now, though.

Reply:
I should remove these since they are beyond the scope of this PR.

Reply:
Did you end up removing the ?

Reply:
Just removed them.